tests/Kernel/fixtures/rendering.yml

```diff
@@ -1,783 +1,94 @@
-- array:
```

Comment:
Instead of creating a `_ecl1` file let's keep one file and comment the tests that should not run. People will then un-comment relevant tests.

Reply:
ok, nice idea.